docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -674,6 +674,35 @@ Default: `256MB`
 
 ---
 
+### Network compression
+
+Use the following two gflags to configure RPC compression:
+
+##### --enable_stream_compression
+
+Controls whether YugabyteDB uses RPC compression. If you set this flag to `true`, you must also specify a stream compression algorithm.
+
+##### --stream_compression_algo
+
+Specifies which compression algorithm to use, if stream compression is enabled. Valid values are:
+
+- 0: no compression
+- 1: gzip
+- 2: snappy
+- 3: lz4
+
+In most cases, LZ4 (`--stream_compression_algo=3`) offers the best compromise of compression performance versus CPU overhead.
+
+{{< note title="Upgrade notes" >}}
+
+To upgrade from an older version that doesn't support RPC compression (such as 2.4), to a newer version that does (such as 2.6), you need to do the following:
+
+1. Rolling restart to upgrade YugabyteDB, such as from 2.4 to 2.6.
```

Comment:
maybe expand on this to make it clear that the role of the upgrade is to get you to a version of the software that is aware of compression, as a feature